Makefile.cache

```diff
@@ -295,13 +295,17 @@ define SAVE_INTO_CACHE
 		echo "Target $(1) dependencies are modifed - global save cache skipped" >> $($(1)_DST_PATH)/$(1).log
 		$(eval $(1)_CACHE_DIR := $(SONIC_DPKG_LOCAL_CACHE_DIR))
 	 )
-	$($(1)_CACHE_USER) tar -C $($(1)_BASE_PATH) -mczvf $($(1)_CACHE_DIR)/$(MOD_CACHE_FILE) $(2) $(addprefix $($(1)_DST_PATH)/,$($(1)_DERIVED_DEBS) $($(1)_EXTRA_DEBS) )  \
+	$($(1)_CACHE_USER) tar -C $($(1)_BASE_PATH) -mcvf $($(1)_CACHE_DIR)/$(MOD_CACHE_FILE).tar $(2) $(addprefix $($(1)_DST_PATH)/,$($(1)_DERIVED_DEBS) $($(1)_EXTRA_DEBS) )  \
 		1>>$($(1)_DST_PATH)/$(1).log
-	sudo chmod 777 $($(1)_CACHE_DIR)/$(MOD_CACHE_FILE)
 
 	echo "File $($(1)_CACHE_DIR)/$(MOD_CACHE_FILE) saved in cache " >> $($(1)_DST_PATH)/$(1).log
 	echo "[ CACHE::SAVED ] $($(1)_CACHE_DIR)/$(MOD_CACHE_FILE)" >> $($(1)_DST_PATH)/$(1).log
 
+	$($(1)_CACHE_USER) tar -C $($(1)_BASE_PATH) -muvf $($(1)_CACHE_DIR)/$(MOD_CACHE_FILE).tar $($(1)_DST_PATH)/$(1).log
+	$($(1)_CACHE_USER) gzip -9 $($(1)_CACHE_DIR)/$(MOD_CACHE_FILE).tar
```

Comment:
I'm not sure about -9, maybe it's better to stick with a default compression level? This change might introduce extra build time.

Reply:
Handled